src/content/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model.mdx

```diff
@@ -42,7 +42,7 @@ A New Relic user is prohibited from sharing their login with other people. A New
 
 ## Updates about our new user model [#migrate]
 
-In July of 2020, we released a new user model called the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models), which offers many [benefits in terms of how you manage your organization and users](https://newrelic.com/blog/how-to-relic/user-management-and-access-security). At first this was only available to new sign-ups but over time we've been migrating more older customers to the new model. Some older customers are able [to migrate their users on their own](/docs/accounts/original-accounts-billing/original-users-roles/user-migration/). We'll continue working on migrating users to the new model until the original model is fully deprecated. 
+In July of 2020, we released a new user model called the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models), which offers many [benefits in terms of how you manage your organization and users](https://newrelic.com/blog/how-to-relic/user-management-and-access-security). At first this was only available to new sign-ups but over time we've been migrating older customers to the new model. Some older customers are able [to self-serve the migration of their users](/docs/accounts/original-accounts-billing/original-users-roles/user-migration/). We'll continue working on migrating users to the new model until the original model is fully deprecated. 
```

Comment:
"self-serve" as a verb seems weird to me, but I can't think of a better, more concise alternative